package.py

```diff
@@ -498,26 +498,28 @@ def __get_versions(self, packages: list[str]) -> dict[str, str]:
         the requirements.txt
         :returns: A Dict where the key is the package and the value is the package version
         """
-        requirements = self.__storage.working_directory / 'requirements.txt'
         package_versions: dict[str, str] = {}
-        with open(requirements, encoding='utf8') as fp:
-            while True:
-                line = fp.readline()
-                if not line:
-                    break
-                if len(line.strip()) == 0 or line.startswith('#'):
-                    continue
-                requirement = line.split('#')[0]
-                req = requirement.split(';')
-                requirement = req[0]
-                if len(req) > 1 and req[1].strip() == "sys_platform == 'win32'":
-                    continue
+        for requirement_file_name in ('requirements.txt', 'requirements_packaging.txt'):
+            requirements = self.__storage.working_directory / requirement_file_name
+            with open(requirements, encoding='utf8') as fp:
+                while True:
+                    line = fp.readline()
+                    if not line:
+                        break
+                    if len(line.strip()) == 0 or line.startswith('#'):
+                        continue
+                    requirement = line.split('#')[0]
+                    req = requirement.split(';')
+                    requirement = req[0]
+                    if len(req) > 1 and req[1].strip() == "sys_platform == 'win32'":
+                        continue
+
+                    split_requirement = requirement.split('==')
+                    package_name = split_requirement[0]
+                    if package_name in packages:
+                        package_version = split_requirement[1]
+                        package_versions[package_name.strip()] = package_version.strip()
```

Comment:
Also here please raise an error if `package_versions` already contains a version for this. This would mean that a file redefines a package.